### DISCUSSION TITLE
Pinned issues don't have labels

### DISCUSSION THREAD

**Discussion by ris58h** [community/community#50681] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Pinned issues should have labels assigned to them.

### Example
Issue in the `Issues` list:
<img width="294" alt="Screenshot 2023-03-22 at 11 45 15" src="https://user-images.githubusercontent.com/511242/226848253-b9492da2-8dff-41b8-bf0b-8b201d268852.png">

The same issue but pinned:
<img width="423" alt="Screenshot 2023-03-22 at 11 45 59" src="https://user-images.githubusercontent.com/511242/226848417-f044f589-e326-4efd-9073-7cfbda874054.png">

The pinned issue doesn't have `bug` label.

**Comment by queenofcorgis** [comment]
Hi there - this is expected behavior, but we're happy to note this feedback, feel free to add any other details or use cases for our Product team.

**Reply by ris58h** [reply]
The `bug` label draws the user's attention to an issue.
Having such indicator in pinned issues helps users to get that something doesn't work as expected.
